.github/CONTRIBUTING.md

```diff
@@ -44,10 +44,19 @@ To make review of PRs easier, please:
 
 ## Commit and PR Messages
 
-* **Reference issues, wiki pages, and pull requests liberally!**
+The [Angular style](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)
+**is enforced for at least one commit in every PR** to make the automatic semantic versioning work.
+For example, a commit might look like:
+
+`feat(#0000): your commit message here`
+
+where feat is the commit type, the options are feat|fix|docs|style|refactor|perf|test|chore
+and 0000 is the Github issue number.
```

Comment:
I'd suggest using `xxxx` instead `0000` here. `0000` is a possible valid input, where xs force the user to input the ticket number.